README.md

```diff
@@ -1355,6 +1355,23 @@ subject.rescue_from Grape::Exceptions::ValidationErrors do |e|
 end
 ```
 
+Grape returns all validation and coercion errors found by default. To skip all subsequent validation checks when a specific param is found invalid, use `fail_fast: true`.
+For example:
+```ruby
+params do
+  required :beer, fail_fast: true
+  required :wine
+end
+```
+will not check if `:wine` is present unless it finds `:beer`. The result of empty params would be a single `Grape::Exceptions::ValidationErrors` error.
```

Comment:
I think a lot of grape examples try to make full sentences, so prefer "The following example will not ..." to having "For example: [code] will not".